apps/condo/domains/acquiring/utils/testSchema/index.js

```diff
@@ -183,10 +183,24 @@ async function makeAcquiringContextAndIntegrationManager() {
     }
 }
 
+async function addAcquiringIntegrationAndContext(client, organization, allowedBillingIntegrationIds = []) {
```

Comment:
I think it's not obvious that the function starting with `add` returns a value

Reply:
on the other hand its usable because it returns the entities which had been added

I dont know of a better alternative -- if you come up with something - please tell